motr/client.h

```diff
@@ -579,7 +579,12 @@ enum m0_op_obj_flags {
 	 * Write, alloc and free operations wait for the transaction to become
 	 * persistent before returning.
 	 */
-	M0_OOF_SYNC   = 1 << 1
+	M0_OOF_SYNC   = 1 << 1,
+        /**
+         * This flag is to describe that this write is from clovis/motr 
+         * application and not from s3.
+         */
+        M0_OOF_MOTR_APP = 1<<2
```

Comment:
Alignment issue for the new code.

Also, add a space before and after operator.
```
M0_OOF_MOTR_APP = 1 << 2
```

Reply:
okay. Will fix it.